public/index.html

```diff
@@ -2,7 +2,7 @@
 <html lang="en">
 
 <head>
-    <meta charset="utf-8" />
```

Comment:
Why did you remove this?

Reply:
Done.

Reply:
But I don't know probably it was an accident ' cos I wasn't aware of this one .Sorry